olive/passes/pytorch/gptq.py

```diff
@@ -215,16 +197,17 @@ def get_onnx_quant_linear(*args, **kwargs):
         quantized_model = quantized_model.model
         assert self.accelerator_spec.accelerator_type == Device.GPU
 
-        converted_onnx_model = OnnxConversion._export_pytorch_model(  # pylint: disable=protected-access
-            quantized_model,
-            model.get_dummy_inputs(),
-            model.get_io_config(),
-            config,
-            config["export_device"],
-            None,
-            tempfile.tempdir,
+        output_model_path = normalize_path_suffix(output_model_path, "model.pt")
+        torch.save(quantized_model, output_model_path)
+
+        model_config = model.to_json()["config"]
+        model_config["model_path"] = output_model_path
+        model_config.pop("model_loader")
```

Comment:
![image](https://github.com/microsoft/Olive/assets/13343117/6ee8820b-5795-4d8a-806c-7ca2e7f295f2)
Can we guarantee `model_loader` always exists? Maybe `model_config.pop("model_loader", None)` is better?

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/afc1de11-3bd3-4385-aa45-9136b9e1f255)
Seems for pytorch model handler, the `model_loader` exists.

Reply:
Yeah, default value is missed, it should not raise error if "model_loader" not setted

Reply:
Yeah, default value is missed, it should not raise error if "model_loader" not setted